### PR TITLE
Added new ruby version to build ruby: 2.6-prev3 and update chrome

### PR DESCRIPTION
**Pull request by pftg**

### CODE REVIEW

**Comment by pftg** [comment]
@miry there is a problem to build this image with `make ruby-chrome-build`:

image

**Comment by miry** [comment]
@pftg i have the same issue on my machine. you can check circleci ruby build.

File: Makefile [hunk 1]
```diff
@@ -1,8 +1,8 @@
 KUBE_VERSION=v1.12.2
-RUBY_CHROME_VERSION=2.6.0-preview2-v2
-RUBY_VERSION=2.6.0-preview2
+RUBY_CHROME_VERSION=2.6.0-preview3-v1
+RUBY_VERSION=2.6.0-preview3
 RUBY_MAJOR=2.6-rc
-NODE_VERSION=3.11.0
+NODE_VERSION=3.141.5
```
Comment (miry) [inline Makefile:5]: @pftg dont change for now Node version.

**Comment by miry** [comment]
@pftg did you check node version is working with chrome integration?

**Pull request closed without merging**